gateway/src/templates/vvs.yml

```diff
@@ -0,0 +1,16 @@
+# how much the execution price is allowed to move unfavorably from the trade
+# execution price. It uses a rational number for precision.
+allowedSlippage: '1/100'
+
+# the maximum gas used to estimate gasCost for a pangolin trade.
+gasLimitEstimate: 150688
+
+# how long a trade is valid in seconds. After time passes Mad Meerkat will not
+# perform the trade, but the gas will still be spent.
+ttl: 300
+
+tradingTypes: ['EVM_AMM']
```

Comment:
Pls remove this 
```suggestion
```

Reply:
The tradingTypes shouldn't be configurable. 
We should only add things we want users to be able to change in the configuration files.
Pls remove it and update all other necessary files.